packages/gatsby/src/schema/resolvers.ts

```diff
@@ -449,10 +494,7 @@ function getFieldNodeByNameInSelectionSet(
   )
 }
 
-export const defaultFieldResolver: GatsbyResolver<
-  any,
-  any
-> = function defaultFieldResolver(source, args, context, info) {
+function _defaultFieldResolver(source, args, context, info): any {
```

Comment:
why not keep const & function on same line? Does _defaultFieldResolver still get correct typings?

Reply:
Because then a lint rule of premature usage will trigger, which we do not apply to function declarations.

My attempt failed anyways so I'll change it back and move the whole function instead :shrug: